### PR TITLE
Added Azure CosmosDB module

### PR DESCRIPTION
**Pull request by kedarghule**

### CODE REVIEW

File: cartography/data/jobs/cleanup/azure_database_account_cleanup.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (n:AzureCosmosDBEnabledLocation)<-[:READ_PERMISSIONS_FROM]-(:AzureDatabaseAccount)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (:AzureCosmosDBEnabledLocation)<-[r:READ_PERMISSIONS_FROM]-(:AzureDatabaseAccount)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
```
Comment (mpurusottamc) [inline cartography/data/jobs/cleanup/azure_database_account_cleanup.json:29]: Let's rename AzureCosmosDBEnabledLocation to AzureCosmosDBLocation. Sorry, i should have highlighted it before. These are locations, independent of whether enabled or not.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
Overall code looks good. Just change one of the entity names.

File: cartography/data/jobs/cleanup/azure_database_account_cleanup.json (hunk 1 above)
Comment (kedarghule) [reply cartography/data/jobs/cleanup/azure_database_account_cleanup.json:29]: Alright. Will do!

Comment (kedarghule) [reply cartography/data/jobs/cleanup/azure_database_account_cleanup.json]: Done

**Review by mpurusottamc — APPROVED** [review]
Looks good

**Pull request merged**